docs/how-to/manage-data/load-data/_category_.json

```diff
@@ -0,0 +1,4 @@
+{
+    "label": "Load Data",
```

Comment:
We should add load data via the UI under here

Reply:
I'll do this as a separate PR to not delay ReBAC. I've created all the loading data pages I should cover here.

![image](https://github.com/permitio/docs/assets/109458126/3018b2a0-5146-4676-9402-076c11715595)